doc/configurable.md

```diff
@@ -15,9 +15,13 @@ Here `Configurable` takes the class specified with the `factory` key and
 initializes it using the other items in the dictionary as kwargs.
 Therefore, all parameters not mentioned in the dictionary fallback to the
 defaults.
-In case of nested dictionaries all nested classes with a factory key are first 
-initialized by `Configurable` before using them as input to the upper-level 
-class. 
+In case of nested dictionaries all nested classes with a `factory` key are 
+first initialized by `Configurable` before using them as input to the upper-level 
+class. The value for `factory` may either be class, a function or a string.
```

Comment:
`Callable` in general?

Reply:
I think not everyone is familiar with the name `Callable`. So maybe `Callable (e.g. class or function)`?